setup.py

```diff
@@ -74,7 +74,7 @@
         "Topic :: System :: Monitoring",
         ],
     install_requires=[
-        'requests<=2.10.0',
+        'requests>=0.12.1,<=2.10.0',
```

Comment:
Ah. That is a good point. Then the correct behavior is probably to require >= the min version and throw an exception and disable if the wrong version of requests is used.